board/safety.h

```diff
@@ -114,7 +114,7 @@ static const safety_hooks *current_hooks = &nooutput_hooks;
 safety_config current_safety_config;
 
 static bool is_lat_active(void) {
-  return controls_allowed || mads_is_lateral_control_allowed_by_mads();
+  return (controls_allowed && !mads_system_enabled()) || mads_is_lateral_control_allowed_by_mads();
```

Comment:
```suggestion
  return controls_allowed || mads_is_lateral_control_allowed_by_mads();
```
This is not supposed to happen. Do the whole "decision" if lat is active inside the MADS method instead